palindrome of intiger

```diff
@@ -0,0 +1,29 @@
+#include <bits/stdc++.h>
+using namespace std;
+int palindrome(int n,int b){
+    int s=0;
+     do
+     {
```

Comment:
Remove do, while condition. It hampers code readability

Reply:
ok